File: Exercicio_006.java
import java.util.Scanner;
public class Exercicio_006 {
    public static void main(String[] args) {
        /* EXERCICIO 6: Crie um programa que faça a tabuada do 1 ao 10 do numero que o usuario digitar */

        try (Scanner Input = new Scanner(System.in)) {
            System.out.println("Digite o numero da tabuada que deseja ver: ");
            int Numero = Input.nextInt();

            System.out.println("______________________________________");

            for (int i = 0; i < 11; i++) {
                System.out.println(Numero + " X " + i + " = " + Numero * i);
            }

            System.out.println("______________________________________");

        }

    }

}

File: Exercicios_PYTHON/Exercicio_006.py
#EXERCICIO 6: Crie um programa que faça a tabuada do 1 ao 10 do numero que o usuario digitar

Numero = int(input("Digite o numero da tabuada que deseja ver:"))
print("_______________")

for x in range(1,11):
    print(Numero,"X",x,"=",Numero*x)

print("_______________")
